recipes/dlib/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "dlib" %}
+{% set version = "19.0" %}
+{% set blas_variant = "openblas" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/davisking/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: a6463524cdb1d275ab408f182b0d47af143179fde7f1dd97074dc25f2f9a2676
+
+  patches:
+    - osx_jpeg.patch                # [osx]
+    - lapack_sgetrf.patch           # [unix]
+    - win_boost.patch               # [win]
+    - win_png.patch                 # [win]
+    - add_global_cxx_release.patch
```

Comment:
Have these patches been sent upstream? If so, it would be good to cite the PR/issue link a comment in the recipe. Possibly with a small explanation about each one.